docker/BuildJekyll.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ruby:3.0.0
+
+WORKDIR /srv/jekyll
+COPY ./static/Gemfile .
+
+ARG uid
+ARG uname
+ARG gid
+ARG gname
+
+RUN groupadd -g $gid $gname
+RUN useradd -ms /bin/bash -u $uid -g $gid $uname
+
+RUN gem install jekyll bundler
```

Comment:
Does this depend on the `Gemfile`? I don't think it does, which suggests to me you could be getting a different version of jekyll every time you run this. You probably really want to pin the version of jekyll somehow: `gem install jekyll -v 4.3.3` seems current, per https://stackoverflow.com/a/27563681 & https://rubygems.org/gems/jekyll . I'd suggest the same for bundler (with a different version number, of course).

Also, the way this is written, any changes to the Gemfile will re-install these two gems, because the `COPY ./static/Gemfile .` above will mean the `RUN gem install ...` here "sees" a different filesystem (and the builder conservatively assumes that `gem install` will change its behavior based on any such differences). Consider moving the "static" install bits above the `COPY` to save yourself from having to repeat the same `gem install` more often than you need to.

Reply:
:heart: makes sense!